src/operator/OrgOverlay.tsx

```diff
@@ -221,25 +228,29 @@ const OrgOverlay: FC = () => {
             </Grid>
           </Overlay.Body>
           <Overlay.Footer>
-            <ButtonBase
-              color={ComponentColor.Default}
-              onClick={() => history.goBack()}
-              testID="org-overlay--cancel-button"
-            >
-              Cancel
-            </ButtonBase>
-            <ButtonBase
-              color={ComponentColor.Primary}
-              onClick={updateLimits}
-              testID="org-overlay--submit-button"
-              status={
-                updateLimitStatus === RemoteDataState.Error
-                  ? ComponentStatus.Disabled
-                  : ComponentStatus.Default
-              }
-            >
-              Submit Changes
-            </ButtonBase>
+            {hasWritePermissions() && (
+              <ButtonBase
+                color={ComponentColor.Default}
+                onClick={() => history.goBack()}
+                testID="org-overlay--cancel-button"
+              >
+                Cancel
+              </ButtonBase>
+            )}
+            {hasWritePermissions() && (
+              <ButtonBase
+                color={ComponentColor.Primary}
+                onClick={updateLimits}
+                testID="org-overlay--submit-button"
+                status={
+                  updateLimitStatus === RemoteDataState.Error
+                    ? ComponentStatus.Disabled
+                    : ComponentStatus.Default
+                }
+              >
+                Submit Changes
+              </ButtonBase>
+            )}
```

Comment:
rather than making a request to the same function twice, we can combine these by wrapping it in a react fragment or just assigning the return value to constant somewhere